### PR TITLE
Expose exemptfee in the SDK Config struct

### LINKED ISSUE #438: Expose exempt fee configuration
Currently, the cln `exemptfee` value is hardcoded. It would be useful to have this config exposed similar to the sdks `maxfee_percent` config.

### PR DESCRIPTION
**Pull request by ok300**
Fixes #438

### CODE REVIEW

**Review by JssDWt — APPROVED** [review]
LGTM

**Review by roeierez — CHANGES_REQUESTED** [review]
Looks good. React Native is missing.

**Review by roeierez — APPROVED** [review]
LGTM

File: libs/sdk-react-native/android/src/main/java/com/breezsdk/BreezSDKMapper.kt [hunk 1]
```diff
@@ -15,8 +15,9 @@ fun asConfig(config: ReadableMap): Config? {
         val defaultLspId = config.getString("defaultLspId")
         val apiKey = config.getString("apiKey")
         val maxfeePercent = config.getDouble("maxfeePercent")
+        val exemptfeeMsat = config.getDouble("exemptfeeMsat")
 
-        return Config(breezserver, mempoolspaceUrl, workingDir, asNetwork(network), paymentTimeoutSec.toUInt(), defaultLspId, apiKey, maxfeePercent, nodeConfig)
+        return Config(breezserver, mempoolspaceUrl, workingDir, asNetwork(network), paymentTimeoutSec.toUInt(), defaultLspId, apiKey, maxfeePercent, exemptfeeMsat, nodeConfig)
```
Comment (dangeross) [inline libs/sdk-react-native/android/src/main/java/com/breezsdk/BreezSDKMapper.kt:20]: I think it needs converting to a ULong

File: libs/sdk-react-native/ios/BreezSDKMapper.swift [hunk 2]
```diff
@@ -97,7 +97,8 @@ class BreezSDKMapper {
            let workingDir = config["workingDir"] as? String,
            let networkStr = config["network"] as? String,
            let paymentTimeoutSec = config["paymentTimeoutSec"] as? UInt32,
-           let maxfeePercent = config["maxfeePercent"] as? Double {
+           let maxfeePercent = config["maxfeePercent"] as? Double,
+           let exemptfeeMsat = config["exemptfeeMsat"] as? Double {
```
Comment (dangeross) [inline libs/sdk-react-native/ios/BreezSDKMapper.swift:101]: Can be as UInt64

**Review by dangeross — CHANGES_REQUESTED** [review]

File: libs/sdk-react-native/android/src/main/java/com/breezsdk/BreezSDKMapper.kt (hunk 1 above)
Comment (ok300) [reply libs/sdk-react-native/android/src/main/java/com/breezsdk/BreezSDKMapper.kt]: Done

File: libs/sdk-react-native/ios/BreezSDKMapper.swift (hunk 2 above)
Comment (ok300) [reply libs/sdk-react-native/ios/BreezSDKMapper.swift]: Done

**Review by dangeross — APPROVED** [review]

**Pull request merged**